.github/ISSUE_TEMPLATE/---image-request.md

```diff
@@ -0,0 +1,27 @@
+---
+name: "\U0001F3A8 Image request"
+about: Request a visual for a page
+title: ''
+labels: design
+assignees: ''
+
+---
+
+**Request the visual**
+
+Hello, I would like to request a visual for the ...
+
+**Link to the page**
+
+_Please add a link to the page you're requesting a visual for._
+
+**Visual Type**
+
+What kind of visual are you requesting? (Header image, content image, video, prototype, etc.)
```

Comment:
Prototypes and static visuals could prob each be their own template as they are quite different and will likely be contributed by people with different skills sets (images more graphic designers, prototypes more UI/UX focused designers).

Reply:
Too many templates = confusion. Let's keep it simple. The request template will probably be the same regardless.

Reply:
How's about call this media request which shows it encompasses opening an issue for any kind of media (prototype, image, video etc.)

Reply:
I think the `image` will be 90% of the requests tbh, let's see if it confuses people and if it does, we can, later on, add it as a `media` template. Let's first see how people use it.